### PR TITLE
chore: add 1on1 sample

### PR DESCRIPTION
**Pull request by kannan-xiao4**
OnUnity: image
OnBrowser: 
image

I am checking regression
- Fix test
- Check working on video-player

### CODE REVIEW

File: WebApp/public/scripts/signaling.js [hunk 1]
```diff
@@ -53,7 +58,7 @@ export default class Signaling extends EventTarget {
 
   async loopGetAnswer() {
     // receive answer message from 30secs ago
```
Comment (karasusan) [inline WebApp/public/scripts/signaling.js:60]: Could you fix this comment?

File: WebApp/public/scripts/signaling.js [hunk 2]
```diff
@@ -63,7 +68,11 @@ export default class Signaling extends EventTarget {
       const answers = data.answers;
 
       answers.forEach(answer => {
-        this.dispatchEvent(new CustomEvent('answer', { detail: answer }));
+        const answerData = {
```
Comment (karasusan) [inline WebApp/public/scripts/signaling.js:71]: I would like to know why this is changed.

File: WebApp/public/scripts/signaling.js [hunk 3]
```diff
@@ -79,11 +88,17 @@ export default class Signaling extends EventTarget {
       lastTimeRequest = Date.parse(res.headers.get('Date'));
 
       const data = await res.json();
-      const candidates = data.candidates.filter(v => v.connectionId = this.connectionId);
-
-      if (candidates.length > 0) {
-        for (let candidate of candidates[0].candidates) {
-          this.dispatchEvent(new CustomEvent('candidate', { detail: candidate }));
+      const candidatesList = data.candidates.filter(v => v.connectionId = this.connectionId);
```
Comment (karasusan) [inline WebApp/public/scripts/signaling.js:91]: Is this correct? `v => v.connectionId = this.connectionId`

File: WebApp/public/scripts/signaling.js (hunk 2 above)
Comment (kannan-xiao4) [reply WebApp/public/scripts/signaling.js:71]: I need to include ConnectionId.
This is same data structure as  `DescData` in SignalingMessage.cs on the C# side.

File: WebApp/public/scripts/signaling.js (hunk 3 above)
Comment (kannan-xiao4) [reply WebApp/public/scripts/signaling.js:91]: oh, this is not correct. I fix it.

File: com.unity.template.renderstreaming-hd/Assets/Scripts/OneOnOneSample.cs [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using Unity.RenderStreaming;
+using Unity.RenderStreaming.Signaling;
+using Unity.WebRTC;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class OneOnOneSample : MonoBehaviour
```
[the remark this answers is not in the document]
Comment (kannan-xiao4) [reply com.unity.template.renderstreaming-hd/Assets/Scripts/OneOnOneSample.cs]: I added

**Pull request closed without merging**